app/forms/webauthn_setup_form.rb

```diff
@@ -10,7 +10,7 @@ class WebauthnSetupForm
 
   attr_reader :attestation_response, :name_taken
 
-  def initialize(user, user_session)
+  def initialize(user, user_session, nice_name)
```

Comment:
Two thoughts:

- Personally, I find that unless positional arguments are obvious enough to know what they'd be by their context alone (like with `from_user_agent`), I generally prefer named keyword arguments, especially as the number of arguments grows
- We switch between "nice name" and "device name" in this file, can we pick one?

```suggestion
  def initialize(user:, user_session:, device_name:)
```

Reply:
Yeah though depending on [my other comment below](https://github.com/18F/identity-idp/pull/9657#discussion_r1410712365) maybe we'd just want `name:` ?